config.yaml

```diff
@@ -717,6 +717,16 @@ teams:
       - ryjones
     members:
       - hendrikebbers
+  - name: hashgraph-online-standards-sdk-js-maintainers
+    maintainers:
+      - kantorcodes
+      - rocketmay
+      - hgraphpunks
+    members: []
```

Comment:
pls add rocketmay and hgraphpunks to members. A group only need 1 maintainer.

Reply:
Pefect!